### ISSUE TITLE
Addition of -rst in restart file names results in Cassandra error

### ISSUE THREAD

**Issue by rmatsum836** [MaginnGroup/mosdef_cassandra#89]
**Bug summary**

I was attempting to restart a simulation that had the filename `equil-rst`, which is the default name given by `mosdef_cassandra` when restarting an equilibration run.  Cassandra currently doesn't accept dashes in the filenames which results in an error.  This should be easily fixed by appending `_rst` to filenames instead of `-rst`.

**Code to reproduce the behavior**

```
        custom_args["run_name"] = "equil-rst_continue"
        custom_args["restart_name"] = "equil-rst"
        # Run NVT production
        mc.restart(
        system=system,
        moveset=moves,
        run_type="production",
        run_length=job.sp.nsteps_prod,
        temperature=300.0 * u.K,
        **custom_args,
        )
```
**Expected behavior**

Restart of simulation from `equip-rst.chk` should occur, but instead results in an error.

**Software versions**

Please include the following details:

- Linux OS
- `Cassandra` 1.2.5
- `mosdef_cassandra` 0.2.1
- `mbuild` 0.10.9
- `foyer` 0.7.6

**Comment by rsdefever** [comment]
Related to #75. Let's add error checking to the `run_name` as well as fixing it from `-rst` to `_rst`.

**Comment by rsdefever** [comment]
Closed by #92 and #93.
